pandas/core/internals/blocks.py

```diff
@@ -1348,7 +1348,7 @@ def get_values(self, dtype: DtypeObj | None = None) -> np.ndarray:
         """
         return object dtype as boxed values, such as Timestamps/Timedelta
         """
-        values = self.values
+        values: ArrayLike = self.values
```

Comment:
values is specifically ExtensionArray here (hopefully mypy can figure that out on its own)

Reply:
Yes, but the issue is that a few lines later, we say `values = values.astype(object)`, and `values` is an `np.ndarray` in this case, so by declaring it as `ArrayLike`, we cover that case.